packages/admin/cms-admin/src/dam/config/useDamScope.ts

```diff
@@ -1,9 +1,17 @@
+import * as React from "react";
+
 import { useContentScope } from "../../contentScope/Provider";
+import { DamScopeContext } from "./DamScopeContext";
 import { useDamConfig } from "./useDamConfig";
 
 function useDamScope(): Record<string, unknown> {
     const { scopeParts = [] } = useDamConfig();
     const { scope: completeScope } = useContentScope();
+    const explicitDamScope = React.useContext(DamScopeContext);
+
+    if (explicitDamScope) {
+        return explicitDamScope;
+    }
 
     return scopeParts.reduce((damScope, scope) => {
```

Comment:
could you move this scopeParts extraction into DamPage and *always* have a DamScopeContext? useDamScope would be simply React.useContext(DamScopeContext) then.

Reply:
ok, done in 7f18d4290485daba180ffd0388f1778be617f031